.github/workflows/nodejs.yml

```
@@ -28,6 +28,7 @@ jobs:
         exit 0
     - name: Install dependencies (apt)
       run: |
+        sudo apt-get update;
```

This semicolon is not needed as `|` already means preserve newlines (see <https://yaml-multiline.info/>).

```suggestion
        sudo apt-get update
```

Thanks for helping to fix the CI!